docs/typescript/introduction.md

```diff
@@ -93,31 +106,35 @@ If you want to run Temporal without Docker, DataDog has created an experimental
 
 ### Step 1: Create a new project
 
-Use the [package initializer](./package-initializer) to create a new project:
+Use the [package initializer](/docs/typescript/package-initializer) to create a new project:
 
 ```bash
 npx @temporalio/create@latest ./example
 cd example
 ```
 
-This will set up with [the basic Hello World sample](https://github.com/temporalio/samples-typescript/tree/main/hello-world).
-If you want to start from a different sample, pass the `--sample <sample-name>` argument to the script. For example:
-
-- running `npx @temporalio/create@latest ./example --sample hello-world-mtls` will download the [sample for connecting to a Temporal Server instance secured with mTLS](https://github.com/temporalio/samples-typescript/tree/main/hello-world-mtls)
-- running `npx @temporalio/create@latest ./example --sample fetch-esm` will download the [sample for using pure ESM Node Modules in activities](https://github.com/temporalio/samples-typescript/tree/main/fetch-esm) ([different configs are needed](https://github.com/temporalio/samples-typescript/tree/main/fetch-esm#fetch-esm))
+This will set up with [the basic Hello World sample](https://github.com/temporalio/samples-typescript/tree/main/hello-world) using our [Package Initializer](/typescript/package-initializer) (think of it like `create-temporal-app`!)
```

Comment:
```suggestion
This will set up with [the basic Hello World sample](https://github.com/temporalio/samples-typescript/tree/main/hello-world) using our [Package Initializer](/docs/typescript/package-initializer) (think of it like `create-temporal-app`!)
```